src/main/java/org/primeframework/mvc/http/FormDataBodyHandler.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2016-2022, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.primeframework.mvc.http;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Brian Pontarelli
+ */
+public class FormDataBodyHandler {
```

Comment:
I did not implement the native `java.net.http.*` interfaces (e.g. `BodyHandler` and `BodyPublisher`) that deal with this or the responses (see `JSONResponseHandler`) since I am less familiar with them. I just kept the same code from Restify, excluding what was no longer needed, and tied together